nss-day-cohort-45/nutyshell-talking-with-a-full-mouth-1
# DOM only renders tasks that are not complete

# Description
Click event added to taskHTMLRep so that the we can listen for the click on the checkbox. checkTaskAsComplete added to taskProvider to change the value of complete from false to true so that it can be taken off of the DOM. 
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Press create new task button and create a new task. The "complete" key in the api should have a value of false. Click the checkbox and see if it is taken off the DOM but still remains in the json file. Now the "complete" key should have a value of true.
# Checklist:
